src/main/java/com/salesforce/storm/spout/dynamic/coordinator/SpoutCoordinator.java

```diff
@@ -146,7 +146,7 @@ public SpoutCoordinator(
 
         // Create new ThreadFactory
         final ThreadFactory threadFactory = new ThreadFactoryBuilder()
-            .setNameFormat("[DynamicSpout] Pool %d on " + threadContext.toString() + " ")
+            .setNameFormat("[DynamicSpout:Coordinator] VirtualSpout Pool %d on " + threadContext.toString() + " ")
```

Comment:
Can we just call this SpoutCoordinator instead of DynamicSpout:Coordinator?

Reply:
I'd prefer to keep them all with a common prefixed.  It makes it way easier to see what's spawned from our stuff when its all grouped together (sorted alphabetically) vs having to hunt through a ton of threads.

Reply:
Fair enough, can we drive that off the class name then?